from data.data import Datos
import custom_exceptions

class DatosHorario(Datos):
    pass